### PR TITLE
fix(semrelease): Provide proper git message for semantic release

### PR DESCRIPTION
**Pull request by ehdsouza**
* Updated `.releaserc` with the proper updates from @padamstx 's PR in core.
* Removed the `[skip ci]` commit message which I had added as a trial
* update travis.yml with `pip3`

### CODE REVIEW

**Comment by padamstx** [comment]
One question I had is...  Appveyor is a CI/CD tool sort of like Travis.  Just wondering why this project is built on both Travis AND Appveyor???

**Comment by ehdsouza** [comment]
@padamstx Appveyor checks against windows

File: test/integration/test_speech_to_text_v1.py [hunk 1]
```diff
@@ -142,11 +142,13 @@ def test_custom_grammars(self):
             assert get_grammar_result is not None
         else:
             print('Deleting grammar')
-            delete_grammar_result = self.speech_to_text.delete_grammar(
-                customization_id,
-                'test-add-grammar-python'
-                ).get_result()
-            assert delete_grammar_result is not None
+            try:
+                delete_grammar_result = self.speech_to_text.delete_grammar(
```
Comment (germanattanasio) [inline test/integration/test_speech_to_text_v1.py]: Codacy Issue found: [local variable 'delete_grammar_result' is assigned to but never used](https://app.codacy.com/app/watson-developer-cloud/python-sdk/pullRequest?prid=4545981)

**Comment by germanattanasio** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/python-sdk/python-sdk/pullRequest?prid=4545981&bid=15295205)

File: test/integration/test_speech_to_text_v1.py (hunk 1 above)
Comment (germanattanasio) [inline test/integration/test_speech_to_text_v1.py]: Codacy Issue found: [Unused variable 'delete_grammar_result'](https://app.codacy.com/app/watson-developer-cloud/python-sdk/pullRequest?prid=4545981)

**Pull request merged**